src/proxy/socks5.rs

```diff
@@ -131,26 +130,56 @@ impl Socks5 {
         .await
     }
 }
-
+async fn handle_socks_connection(mut oc: OutboundConnection, mut stream: TcpStream) {
+    match negotiate_socks_connection(&oc.pi, &mut stream).await {
+        Ok(target) => {
+            // In many cases, when we are fail we don't have this.
```

Comment:
What is _this_ in the comment? We don't have a target?

Reply:
oh, I think this was meant to go refer to the dummy address and is part of a copy/paste from `send_error`